docs/operators/jdbcPOJOInsertOutputOperator.md

```diff
@@ -0,0 +1,154 @@
+JDBC Transactional POJO Output Operator
+==============
+
+## Operator Objective
+This operator receives stream of POJOs as input and using JDBC connection will insert the POJOs in a database table where each POJO corresponds to a row in the database table. The main features of this operator are persisting data to the database table and fault tolerance. This operator creates a transaction at the start of window, executes batches of sql updates, and closes the transaction at the end of the window. Each tuple corresponds to an SQL update statement. The operator groups the updates in a batch and submits them with one call to the database. Batch processing improves performance considerably. The size of a batch is configured by batchSize property. The tuples in a window are stored in check-pointed collection which is cleared in the endWindow(). This is needed for the recovery. The operator writes a tuple exactly once in the database, which is why only when all the updates are executed, the transaction is committed in the end window call. To execute/fire the insert queries on the database table, this operator uses PreparedStatement, a precompiled SQL statement.
```

Comment:
receives stream => receives a stream

Both "insert" and "update" are mentioned in the text here and the semantics need to be clarified
as to what happens if a row for a tuple already exists: will it fail or use some sort of "upsert" query ?

Seems like too much implementation detail for a first paragraph; suggest splitting out the
implementation details into a separate section named "Implementation Notes" or something
similar. This introductory paragraph can just be something like:
"This operator receives an input stream of POJOs and inserts them as rows in a database table in
a fault-tolerant way."

Suggest using short lines instead of one very long line since it allows greater precision for review comments.